salt/metalk8s/repo/Ubuntu.sls

```diff
@@ -0,0 +1,30 @@
+Install python-apt:
+  pkg.installed:
+    - name: python-apt 
+
+
+Configure Kubernetes repository:
```

Comment:
Why do we skip a random number of lines?

- 2 lines between `Install python-apt` and `Configure Kubernetes repository`
- 3 lines between `Configure Kubernetes repository` and `Configure Salt repository`
- 1 line between `Configure Salt repository` and `Repositories configured`.

Reply:
Empty lines deleted.